pkg/pkg.ml

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ocaml
+#use "topfind"
+#require "topkg"
+open Topkg
+
+let lacaml = Conf.with_pkg "lacaml"
+let lbfgs = Conf.with_pkg "lbfgs"
+let ocephes = Conf.with_pkg "ocephes"
+let kaputt = Conf.with_pkg "kaputt"
+let bisect_ppx = Conf.with_pkg "bisect_ppx"
+let coverage = Conf.with_pkg ~default:false "coverage"
+let documentation = Conf.with_pkg ~default:false "documentation"
+
+let full c =
+  let lacaml = Conf.value c lacaml in
+  let lbfgs = Conf.value c lbfgs in
+  let ocephes = Conf.value c ocephes in
+  lacaml && lbfgs && ocephes
+
+let packages ps =
+  List.fold_left (fun acc p -> p :: "-package" :: acc) [] ps
+  |> List.rev
+  |> Cmd.of_list
+
+let () =
+  let build =
+    let cmd c os fpaths =
+      let full     = full c in
+      let coverage = Conf.value c coverage in
+      let is_test  = Conf.pkg_name c = "omltest" in
+      let scmd =
+        Cmd.(Pkg.build_cmd c os
+          %% (of_list ["-plugin-tag"; "package(str)"])       (* Str in myocamlbuild.ml *)
+          %% on full      (packages [ "ocephes"; "lacaml"; "lbfgs"])
```

Comment:
I wouldn't specify packages on the cli. It is harmless to have them in your `_tags` file if you don't use them/they are not installed. This will fail the linting done by `topkg lint` between the `_tags` file and the `opam` file.  (though you can [disable](http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALopam_file) that lint if needed).